src/main/java/mate/controller/GetAllManufacturersController.java

```diff
@@ -0,0 +1,24 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/manufacturers/all")

```

Comment:
![image](https://user-images.githubusercontent.com/98731880/176400459-16885252-e3cd-409d-92c4-1e5915b1583b.png)

Reply:
this is normal url. if i misunderstood you, please explain yourself.